SherpaFrontEnd/Pages/SurveyQuestions/SurveyQuestions.razor

```diff
@@ -136,6 +137,16 @@
         var questionHasFourAnswers = question.Responses[Languages.ENGLISH].Length == 4;
 
 
+        if (question is { Component: HackmanComponent.ExpertCoaching, Subcomponent: HackmanSubComponent.Overall } && _selectedLanguage == Languages.ENGLISH)
+        {
+            return new[] { "Very unhelpful", "Unhelpful", "Neutral", "Helpful", "Very helpful" };
+        }
+
+        if (question is { Component: "Expert Coaching", Subcomponent: "Overall" } && _selectedLanguage == Languages.SPANISH)
```

Comment:
Can we also use the HackmanComponent and HackmanSubComponent constants here instead of hardcoded strings?

Reply:
This has been updated to include the constants